File: module07/module39/module2/module2/src/main/java/com/android/example/module07_module39_module2_module2/ClassAAB.java
package com.android.example.module07_module39_module2_module2;

public class ClassAAB {
	private 	com.android.example.module07_module39_module2_module1.ClassAAB instance_var_1_0 = new com.android.example.module07_module39_module2_module1.ClassAAB();
	private 	com.android.example.module07_module39_module2_module1.ClassAAH instance_var_1_1 = new com.android.example.module07_module39_module2_module1.ClassAAH();

	public void method0(
		com.android.example.module07_module39_module2_module5.ClassAAD param0,
		com.android.example.module07_module39_module2_module1.ClassAAI param1,
		com.android.example.module07_module39_module2_module5.ClassAAD param2,
		com.android.example.module07_module39_module2_module1.ClassAAC param3) throws Throwable {
		for (int iAb = 0; iAb < 2; iAb++) {
			param0.method2(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.internal.schedulers.IoScheduler());

		}
	}

	public void method1(
		com.android.example.module07_module39_module2_module5.ClassAAD param0,
		com.android.example.module07_module39_module2_module5.ClassAAB param1) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.onError(new java.lang.Throwable());

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.isCancelled();

		param0.method1(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber());

	}

	public void method2(
		com.android.example.module07_module39_module2_module5.ClassAAG param0,
		com.android.example.module07_module39_module2_module5.ClassAAF param1) throws Throwable {
	}

	public void method3(
		com.android.example.module07_module39_module2_module1.ClassAAD param0) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.assertSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.isCancelled();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.assertSubscribed();

		io.reactivex.disposables.SerialDisposable local_var_2_4 = new io.reactivex.disposables.SerialDisposable();
		local_var_2_4.isDisposed();

	}

	public void method4(
		com.android.example.module07_module39_module2_module1.ClassAAH param0,
		com.android.example.module07_module39_module2_module5.ClassAAA param1,
		com.android.example.module07_module39_module2_module1.ClassAAG param2) throws Throwable {
	}
}

File: module22/module01/module07/module2/src/main/java/com/android/example/module22_module01_module07_module2/ClassAAB.kt
package com.android.example.module22_module01_module07_module2

@Suppress("UNUSED_PARAMETER")
class ClassAAB {
	private 	val instance_var_1_0: com.android.example.module22_module01_module07_module1.ClassAAI = com.android.example.module22_module01_module07_module1.ClassAAI()
	private 	val instance_var_1_1: com.android.example.module22_module01_module07_module1.ClassAAG = com.android.example.module22_module01_module07_module1.ClassAAG()
	private 	val instance_var_1_2: com.android.example.module22_module01_module07_module1.ClassAAJ = com.android.example.module22_module01_module07_module1.ClassAAJ()
	private 	val instance_var_1_3: com.android.example.module22_module01_module07_module3.ClassAAG = com.android.example.module22_module01_module07_module3.ClassAAG()
	private 	val instance_var_1_4: com.android.example.module22_module01_module07_module3.ClassAAG = com.android.example.module22_module01_module07_module3.ClassAAG()

	fun method0(
		param0: com.android.example.module22_module01_module07_module1.ClassAAD,
		param1: com.android.example.module22_module01_module07_module3.ClassAAH,
		param2: com.android.example.module22_module01_module07_module3.ClassAAI,
		param3: com.android.example.module22_module01_module07_module3.ClassAAF
	) {
		param0.method0(com.android.example.module22_module01_module07_module3.ClassAAC(), com.android.example.module22_module01_module07_module3.ClassAAA())

		param0.method2(com.android.example.module22_module01_module07_module3.ClassAAE(), com.android.example.module22_module01_module07_module3.ClassAAF(), com.android.example.module22_module01_module07_module3.ClassAAJ())

		for (iAb in 0..3) {
			param0.method1(com.android.example.module22_module01_module07_module3.ClassAAA(), com.android.example.module22_module01_module07_module3.ClassAAA(), com.android.example.module22_module01_module07_module3.ClassAAI())

			val local_var_3_0: io.reactivex.internal.schedulers.IoScheduler = io.reactivex.internal.schedulers.IoScheduler()
			local_var_3_0.size()

		}
		param0.method0(com.android.example.module22_module01_module07_module3.ClassAAC(), com.android.example.module22_module01_module07_module3.ClassAAA())

	}

	fun method1(
		param0: com.android.example.module22_module01_module07_module3.ClassAAG,
		param1: com.android.example.module22_module01_module07_module3.ClassAAA
	) {
		param0.method0("SomeString", "SomeString")

	}

	fun method2(
		param0: com.android.example.module22_module01_module07_module3.ClassAAI,
		param1: com.android.example.module22_module01_module07_module3.ClassAAE,
		param2: com.android.example.module22_module01_module07_module1.ClassAAH
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.onError(kotlin.Throwable())

	}

	fun method3(
		param0: com.android.example.module22_module01_module07_module1.ClassAAG
	) {
		param0.method1(com.android.example.module22_module01_module07_module3.ClassAAA(), com.android.example.module22_module01_module07_module3.ClassAAF(), com.android.example.module22_module01_module07_module3.ClassAAG())

		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.dispose()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.assertNotSubscribed()

	}

	fun method4(
		param0: com.android.example.module22_module01_module07_module3.ClassAAI,
		param1: com.android.example.module22_module01_module07_module1.ClassAAA,
		param2: com.android.example.module22_module01_module07_module3.ClassAAE
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.assertSubscribed()

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.hasSubscription()

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.assertNotSubscribed()

	}
}
